data/org_cluster_map_handler.py

```diff
@@ -0,0 +1,113 @@
+import json
+import logging
+import os
+import getorg
+
+from data.models import Contributor
+
+
+def handle(output_dir=None):
+    logger = logging.getLogger(__name__)
+
+    if not output_dir:
+        logger.debug('output_dir arg not provided! Setting output_dir'
+                     ' to cluster_map/')
+        output_dir = 'cluster_map'
+
+    class Location:
+
+        def __init__(self, longitude, latitude):
+            self.longitude = longitude
+            self.latitude = latitude
+
+    org_location_dict = {}
+
+    for contrib in Contributor.objects.all():
+        if not contrib.login.startswith('testuser') and contrib.location:
+            user_location = json.loads(contrib.location)
+            location = Location(user_location['longitude'],
+                                user_location['latitude'])
+            org_location_dict[contrib.login] = location
+            logger.debug('{} location {} added on map'.format(
+                contrib.login, user_location))
+    getorg.orgmap.output_html_cluster_map(org_location_dict,
+                                          folder_name=output_dir)
+
+    move_and_make_changes_in_files(output_dir)
+
+
+def move_and_make_changes_in_files(output_dir):
```

Comment:
Add a proper documentations for any new methods you add.

E.g. see: https://github.com/coala/community/blob/9a7b5f4a32831771ea3e5d1b9befd8e7311b886d/gamification/process/update.py#L25 or https://github.com/coala/community/blob/9a7b5f4a32831771ea3e5d1b9befd8e7311b886d/gamification/models.py#L84